corehq/apps/users/tasks.py

```diff
@@ -37,11 +42,12 @@ def bulk_upload_async(domain, user_specs, group_specs, location_specs):
     }
 
 @task(rate_limit=2, queue='background_queue', ignore_result=True)  # limit this to two bulk saves a second so cloudant has time to reindex
-def tag_docs_as_deleted(cls, docs, deletion_id):
+def tag_cases_as_deleted_and_remove_indices(domain, docs, deletion_id):
     for doc in docs:
         doc['doc_type'] += DELETED_SUFFIX
         doc['-deletion_id'] = deletion_id
-    cls.get_db().bulk_save(docs)
+    CommCareCase.get_db().bulk_save(docs)
+    _remove_indices_from_deleted_cases_task.delay(domain, [doc['_id'] for doc in docs])
```

Comment:
just double checking that you want to start another async task from within an async task